core/src/replay_stage.rs

```diff
@@ -112,6 +128,12 @@ impl Drop for Finalizer {
     }
 }
 
+struct ReplaySlotFromBlockstore {
+    slot_is_dead: bool,
```

Comment:
nit: `slot_is_dead -> is_slot_dead` prefix with `is` for naming booleans

Reply:
fixed